src/styles/blocks/watch.scss

```
@@ -0,0 +1,15 @@
+.watch {
+  left: 50%;
+  top: 50%;
+  margin-top: -300px;
+  margin-left: -300px;
```

```suggestion
  transform: translate(-50%,-50%);
```
You can achieve it easier like this in order not to hardcore the margins
https://www.w3schools.com/css/tryit.asp?filename=trycss_align_transform